test/e2e/dataplane/tcp_pod_to_service.go

```diff
@@ -22,14 +22,14 @@ func testPod2ServiceTCP(f *framework.Framework) {
 	connectorUUID := string(uuid.NewUUID())
 
 	By("Creating a listener pod in cluster B, which will wait for a handshake over TCP")
-	listenerPod := f.CreateTCPCheckListenerPod(framework.ClusterB, listenerUUID)
+	listenerPod := f.CreateTCPCheckListenerPod(framework.ClusterB, listenerUUID, f.Namespace)
```

Comment:
most of the test cases need to live in the default namespace.

Can we define an extra function CreateTCPCheckListenerPodInNS

then CreateTcpCheckListenderPod will just do --> CreateTCPCheckListenerPodInNs(cluster, listenerUUID, f.Namespace).

and we don't need to change existing tests or clutter new ones with extra parameters.